pattern_library/monkey_utils.py

```diff
@@ -11,14 +13,19 @@
 
 
 def override_tag(
-    register: django.template.Library, name: str, default_html: Optional[str] = None
+    register: django.template.Library,
+    name: str,
+    default_html: typing.Optional[typing.Any] = UNSPECIFIED,
 ):
     """
     An utility that helps you override original tags for use in your pattern library.
     """
 
     original_tag = register.tags[name]
 
+    # Save the caller for the override tag in case it's needed for error reporting.
+    trace = inspect.stack()[1]
```

Comment:
If I'm not wrong, calling `inspect.stack()` has a non-negligible cost in terms of performance?  
I wonder if we could call it only when we need it, before the calls to ` warnings.warn`, rather than calling it even when we're staying on the happy path, for which we won't need it? :slightly_smiling_face:

Reply:
Good point: I've now moved this as suggested